rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml

```diff
@@ -20,11 +21,17 @@ detection:
         CommandLine|contains:
             - ' /list '
             - ' list '
-    selection_all:
+    selection_cmd1:
         CommandLine|contains|all:
             - ' /text'
             - ' password'
-    condition: all of selection*
+    selection_cmd2:
+        CommandLine|contains:
+            - ' /@t '
+            - ' /text '
+            - ' /config '
+            - ' /@c'
```

Comment:
I suggest combining both selection_cmd as it's not necessary to separate them anymore.

```yml
selection_cmd:
    CommandLine|contains:
        - ' /@t' # Covers both "/@text:*" and "/@t:*"
        - ' /text'
        - ' /show'
        - ' /config'
```